schema.sql

```diff
@@ -14,3 +14,27 @@ CREATE TABLE animals (
 -- Day 2
 ALTER TABLE animals
 ADD species VARCHAR(100);
+
+-- Day 3
+CREATE TABLE OWNERS (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
```

Comment:
- [ ] Great job working on this project, but as per the project requirements, the `data type` for `id` should be `int` and set to autoincrement.  This I think can do that:
`id int NOT NULL IDENTITY(1,1)`
The data type `Bigserial` is going to reserve too much space to store the id values which can never be used. Kindly fix this.